Package.swift

```diff
@@ -13,28 +19,32 @@ let package = Package(
         .library(name: "AsyncKit", targets: ["AsyncKit"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-nio.git", from: "2.61.0"),
+        .package(url: "https://github.com/apple/swift-nio.git", from: "2.89.0"),
         .package(url: "https://github.com/apple/swift-log.git", from: "1.5.3"),
         .package(url: "https://github.com/apple/swift-collections.git", from: "1.0.5"),
         .package(url: "https://github.com/apple/swift-algorithms.git", from: "1.1.0"),
+        .package(url: "https://github.com/PassiveLogic/swift-dispatch-async.git", from: "0.0.1"),
```

Comment:
Note, I'm working on cutting a semver-stable 1.0.0 release of this. Will update once that is done.

Reply:
@0xTim This now relies on a semver-stable release of `swift-dispatch-async. Please let me know if there is anything else I can do to improve this PR. 🎉